modules/Samples/Samples/Component/Data/Repository/DataSource/BinDataSource.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  BinDataSource.swift
+//  Samples
+//
+//  Copyright 2022 Thoughtworks, Inc. All rights reserved
+//
+
+import Foundation
+import Combine
+
+public protocol BinDataSource {
```

Comment:
How about rename it to `NetworkDataSource`

Reply:
done